settings.gradle

```diff
@@ -1,3 +1,46 @@
+pluginManagement {
+  repositories {
+    google()
+    gradlePluginPortal()
+    mavenCentral()
+  }
+}
+
+dependencyResolutionManagement {
+  versionCatalogs {
+    if (System.getenv("DEP_OVERRIDES") == "true") {
```

Comment:
Can you add a helper method for handling the overrides stuff so that we don't need to duplicate it? I think that would be possible if we defined it in buildSrc? 🤔

Reply:
you can't share logic in pluginManagement or dependencyResolutionManagement blocks unfortunately :/